database/src/main/java/com/firebase/ui/database/FirebaseArray.java

```diff
@@ -20,25 +20,26 @@
 import com.google.firebase.database.Query;
 
 import java.util.ArrayList;
-import java.util.List;
+import java.util.HashMap;
 
 /**
  * This class implements an array-like collection on top of a Firebase location.
  */
 class FirebaseArray implements ChildEventListener {
-    public interface OnChangedListener {
+    interface OnChangedListener {
```

Comment:
Yeah that's true but Studio is not really considering that this is a library for public consumption, so people may want to subclass etc.